includes/Utilities/BackgroundJobCleanup.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Utilities;
+
+use SkyVerge\WooCommerce\Facebook\Heartbeat;
```

Comment:
```suggestion
use SkyVerge\WooCommerce\Facebook\Utilities\Heartbeat;
```

Heartbeat class has moved as you suggested.

Reply:
Doh! Thanks for catching that.